guides/doc-Provisioning_Guide/topics/configuring_the_discovery_service.adoc

```diff
@@ -0,0 +1,44 @@
+[[configuring-the-discovery-service]]
+== Configuring the Discovery Service
+
+{ProjectName} provides a method to automatically detect hosts on a network that are not in your {Project} inventory. These hosts boot the discovery image that performs hardware detection and relays this information back to {ProjectServer}. This method creates a list of ready-to-provision hosts in {ProjectServer} without needing to enter the MAC address of each host.
+
+When you boot a blank, bare-metal host, the boot menu has two options: `local` and `discovery`. If you select `discovery` to boot the Discovery image, after a few minutes, the Discovery image completes booting and a status screen is displayed.
+
+The Discovery service is enabled by default on {ProjectServer}. However, the default setting of the global templates is to boot from the local hard drive. To change the default setting, in the Satellite web UI, navigate to *Administer* > *Settings*, and click the *Provisioning* tab. Locate the *Default PXE global template entry* row, and in the *Value* column, enter *discovery*.
+
+image::PXE-mode.png[]
+
+To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
+
+
+* `tfm-rubygem-foreman_discovery`
+ifeval::["{build}" == "satellite"]
+* `foreman-discovery-image`
+endif::[]
+* `rubygem-smart_proxy_discovery`
+
+ifeval::["{build}" == "satellite"]
+The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
```

Comment:
This sentence is actually valid for all scenarios, only the `foreman-discovery-image` package is the one that needs to be handled differently.